app/schemas/models/interactives/interactive.schema.js

```diff
@@ -7,7 +7,6 @@ const schema = require('../../schemas')
 
 const interactiveSchema = {
   type: 'object',
-  required: ['interactiveType', 'promptText', 'interactiveData', 'unitCodeLanguage'],
```

Comment:
Why was this removed?

Reply:
I created the server APIs keeping the editor in mind. From the editor, I want to allow creating a new interactive with just a name, therefore removed these from the required properties.